modules/test/playwright/tests/analytics-settings-web/connectionInformation.spec.ts

```diff
@@ -0,0 +1,110 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {dataApiHelpersTest} from '../../fixtures/dataApiHelpersTest';
+import { featureFlagsTest } from '../../fixtures/featureFlagsTest';
+import {loginAnalyticsCloudTest} from '../../fixtures/loginAnalyticsCloudTest';
+import {loginTest} from '../../fixtures/loginTest';
+import getRandomString from '../../utils/getRandomString';
+import { PROPERTY_COMMERCE_CHANNEL_COLUMN_INDEX, PROPERTY_SITE_COLUMN_INDEX, expectPropertyColumn, goToSettingsStep, syncAnalyticsCloud, syncCommerce, syncSite } from './utils/analyticsSettings';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	dataApiHelpersTest,
+    featureFlagsTest({
+		'LPS-178052': true,
+	}),
+	loginAnalyticsCloudTest(),
+	loginTest()
+);
+
+test(
+	'Modify the sites and channels synchronized in the property review sidebar',
+	{
+		tag: '@Legacy',
```

Comment:
Hey @marcosapmf, the correct tag should be `LRAC-11044`. You can see this information at the beginning of the `@description` tag in the Poshi test code ([Here](https://github.com/liferay-ac/liferay-portal/pull/1717/files#diff-ef61baf25231b1dcc39ad9212b3c128731413ebd0fbbeb79f4003d72882a1faeL212))